ext/plugins/internal/remove-configuration-html-comments.js

```diff
@@ -20,6 +20,7 @@ module.exports = function() {
 
           if(AstNodeInfo.isConfigurationHtmlComment(bodyEntry)) {
             // remove the entry
+            // BUG: GJ: this remove the correct comment node, but the comment remains in the output for some reason
             node.body.splice(i, 1);
```

Comment:
@rwjblue perhaps you have some insight to lend here? The unit tests for this plugin pass, the comment node is remove correctly here, this plugin is run yet the final output still contains the `<!-- template-lint disable=true -->` comment.